src/components/views/auth/PasswordLogin.js

```diff
@@ -243,9 +245,9 @@ class PasswordLogin extends React.Component {
     render() {
         const Field = sdk.getComponent('elements.Field');
 
-        let forgotPasswordJsx;
+        let forgotPasswordJsx = null;
 
-        if (this.props.onForgotPasswordClick) {
+        if (!SdkConfig.get().disable_password_change && this.props.onForgotPasswordClick) {
```

Comment:
We try to name our new settings in `camelCase` and with positive-sounding names (therefore this would be `passwordChanges: true` as a default).